tests/test_csc.py

```diff
@@ -18,6 +20,34 @@
 
 index_gen = salobj.index_generator()
 
+logger = logging.getLogger()
+logger.level = logging.DEBUG
+
+
+def float_to_str(ang, sep=":"):
```

Comment:
This can be replaced by:
```
from astropy.coordinates import Angle
import astropy.units as u
...
Angle(ang, u.deg).to_string(unit=u.deg, sep=':')
```